unstructured/ingest/main.py

```diff
@@ -385,6 +386,22 @@ def run(self):
     default=None,
     help="Number of days to go back in the history of discord channels, must be an number",
 )
+@click.option(
+    "--elasticsearch-url",
+    default=None,
+    help='URL to the Elasticsearch cluster, e.g. "http://localhost:9200"',
+)
+@click.option(
+    "--elasticsearch-index-name",
+    default=None,
+    help="Name for the elasticsearch index to pull data from",
+)
+@click.option(
+    "--jq-query",
+    default=None,
+    help="JQ query to get a subset of the fields from a JSON document, before processing it. \
+        For a group of JSON documents, it assumes that all of the documents have the same schema",
```

Comment:
do we imagine jq-query being used in other connectors? (I'm guessing so) if _not_ we should prefix with elasticsearch. but assuming so, let's go ahead and indicate that this is currently supported only by the elasticsearch connector (so folks don't try to use with other connectors)

Reply:
Makes sense. If there are other popular stores that represent documents in JSON format I think it'd be beneficial, or even necessary for them. I'll add the note that this is currently only supported in elasticsearch.

Reply:
yep, totally agree

Reply:
Changes applied! (help message updated, saying this is currently only supported for elasticsearch)